conf/messages.en

```diff
@@ -156,8 +156,44 @@ resultsPage.smallProfitRate = Small profit rate
 resultsPage.effectiveCorporationTaxAfterMarginalRelief = Effective Corporation Tax rate after Marginal Relief
 resultsPage.effectiveCorporationTax = Effective Corporation Tax rate
 resultsPage.reducedFromAfterMR = Reduced from {0} after Marginal Relief
-resultsPage.2years.period.heading = Your accounting period covers 2 HMRC financial years
+resultsPage.2years.period.heading = Your account period covers 2 HMRC financial years
```

Comment:
I might have merged incorrectly here. What do you think?

Reply:
Yes, should be "accounting"